server/files/log_file.go

```diff
@@ -33,7 +33,14 @@ func GetLogFile(logName string) (*os.File, error) {
 			path = filepath.Join(usr.HomeDir, gameLogsDir, fileName)
 		}
 	} else {
-		path = filepath.Join("var", "log", fileName)
+		dirPath := filepath.Join("/var", "log", gameLogsDir)
+		if _, err := os.Stat(dirPath); os.IsNotExist(err) {
+			err = os.MkdirAll(dirPath, 0755)
+			if err != nil {
+				fmt.Println("Failed to make nix log directory: ", err)
+			}
+		}
+		path = filepath.Join("/var", "log", gameLogsDir, fileName)
```

Comment:
I thought it might be easier to store log files on the server under their own directory.  So I had it check for the directory's existence and create it if it didn't exist.

Reply:
you can simplify this by doing `filepath.Join(dirPath, fileName`